core/src/test/java/io/fabric8/maven/core/util/KubernetesResourceUtilTest.java

```diff
@@ -92,7 +91,7 @@ public void containsKind() throws Exception {
     public void job() throws Exception {
         HasMetadata ret = getResource(DEFAULT_RESOURCE_VERSIONING, new File(fabric8Dir, "job.yml"), "app");
         assertEquals("Job", ret.getKind());
-        assertEquals(JOB_VERSION, ret.getApiVersion());
+        assertEquals(KubernetesResourceUtil.JOB_VERSION, ret.getApiVersion());
```

Comment:
Why use junit asertions when the project already imports assertj which makes writing and reading assertions better. I am not saying to change all assertions, butmaybe the new tests could start using it.